src/components/form/form.js

```diff
@@ -0,0 +1,182 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import SingleChoice from "./singleChoice";
+import "./form.scss";
+import { Button } from "../button/button";
+import Results from "./results";
+import _ from "underscore";
```

Comment:
I would try to import only what we need, and not everything. This way we reduce the amount of bundled JavaScript we export. Something like this. Not really familiar with underscore, but I know other libraries like `lodash` have it working this way.

```js
import groupBy from "underscore/groupBy";
import mapObject from "underscore/mapObject";
...etc
```